datatypes/src/commonMain/kotlin/at/asitplus/crypto/datatypes/CryptoPublicKey.kt

```diff
@@ -327,18 +343,16 @@ sealed class CryptoPublicKey : Asn1Encodable<Asn1Sequence>, Identifiable {
         fun toAnsiX963Encoded(): ByteArray =
             when (useCompressedRepresentation) {
                 true -> {
-                    val prefix = (2U + compressY().toUInt()).toByte()
-                        .also { require(it == ANSI_COMPRESSED_PREFIX_1 || it == ANSI_COMPRESSED_PREFIX_2) }
                     byteArrayOf(
-                        prefix,
-                        *x.ensureSize(curve.coordinateLengthBytes)
+                        ANSIECPrefix.forSign(yCompressed).prefixByte,
```

Comment:
@JesusMcCloud here it would be very nice if i could just write `ANSIECPrefix.forSign(yCompressed)`